scripts/pi-hole/php/footer.php

```diff
@@ -99,6 +99,10 @@
 
 </div>
 <!-- ./wrapper -->
+
+<?php if ($scriptname === "login") { ?>
+  <script src="scripts/pi-hole/js/login.js"></script>
+<?php } ?>
 <script src="scripts/pi-hole/js/footer.js"></script>
```

Comment:
We need to check if we need footer.js in the login page; if not we should not include it.